app/app.go

```diff
@@ -748,7 +748,7 @@ func NewStrideApp(
 		capability.NewAppModule(appCodec, *app.CapabilityKeeper, false),
 		feegrantmodule.NewAppModule(appCodec, app.AccountKeeper, app.BankKeeper, app.FeeGrantKeeper, app.interfaceRegistry),
 		crisis.NewAppModule(app.CrisisKeeper, skipGenesisInvariants, app.GetSubspace(crisistypes.ModuleName)),
-		ccvgov.NewAppModule(appCodec, app.GovKeeper, app.AccountKeeper, app.BankKeeper, IsProposalWhitelisted, app.GetSubspace(govtypes.ModuleName), func(_ string) bool { return true }),
+		ccvgov.NewAppModule(appCodec, app.GovKeeper, app.AccountKeeper, app.BankKeeper, IsProposalWhitelisted, app.GetSubspace(govtypes.ModuleName), IsModuleWhiteList),
```

Comment:
what's your thinking on using passing in a fn that returns true vs. having the `IsProposalWhitelisted` in another file that return true? the former approach seems slightly simpler to me, but I don't feel strongly!

Reply:
I think either is fine! 

My thinking was - when we were tracing through this together, it took us awhile to realize this function was passed in here cause we couldn't find it via global search. So I figured this gives us a bit more traceability / is a little more explicit!

Reply:
Makes sense to me